video/out/wayland_common.c

```diff
@@ -1031,7 +1012,7 @@ int vo_wayland_init(struct vo *vo)
         .vo = vo,
         .log = mp_log_new(wl, vo->log, "wayland"),
         .scaling = 1,
-        .wakeup_pipe = {-1, -1},
+        .wakeup_pipe = {-1, -1, -1},
```

Comment:
I don't see where wakeup_pipe[2] is ever set to a valid FD?

Reply:
I'll get rid of this.